run_integration_test.go

```diff
@@ -14,11 +14,13 @@ import (
 )
 
 const (
-	tmpdir = "tmp"
-	secret = "password"
-	key    = "user"
-	repopw = "password"
-	repo   = "s3:http://localhost:9000/test"
+	tmpdir        = "tmp"
```

Comment:
Why using a hardcoded path for a temporary directory? I suggest to use https://golang.org/pkg/io/ioutil/#TempDir instead.

Reply:
Didn't know that one yet, makes things much easier :)